Dockerfile

```diff
@@ -1,19 +1,31 @@
 FROM php:7-cli
 
-ENV DEBIAN_FRONTEND noninteractive
+ARG COMPOSER_FLAGS="--prefer-dist --no-interaction"
+ARG DEBIAN_FRONTEND=noninteractive
 ENV COMPOSER_ALLOW_SUPERUSER 1
+ENV COMPOSER_PROCESS_TIMEOUT 3600
 
-WORKDIR /tmp/
+WORKDIR /code/
+
+COPY docker/php-prod.ini /usr/local/etc/php/php.ini
+COPY docker/composer-install.sh /tmp/composer-install.sh
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
 	    git \
         zlib1g-dev \
```

Comment:
Proč jsi to neudělal rovnou copy&paste z template? 

![image](https://user-images.githubusercontent.com/642928/42892214-918ba9c2-8ab1-11e8-9112-fd5a0bbf560f.png)